### ISSUE TITLE
Use of red & green to discriminate failure/success

### ISSUE THREAD

**Issue by craigjh** [18F/web-design-standards#337]
I'd like to point out that there is almost no difference in the colors of Text Entry Error and Text Entry Success for those of us with red-green color vision deficiencies.

**Comment by maya** [comment]
@craigjh great point! How about if we go to our lighter version of green for success - it's 20% lighter than the current. I think red should be darker bc it's stronger and will stand out more.

Light green:

![screen shot 2015-08-24 at 5 50 37 pm](https://cloud.githubusercontent.com/assets/5249443/9456157/dae50094-4a88-11e5-9cea-91605e934744.png)

Green (current):

![screen shot 2015-08-24 at 5 52 26 pm](https://cloud.githubusercontent.com/assets/5249443/9456164/e7f01080-4a88-11e5-8b38-e0295b2175e6.png)

**Comment by mollieru** [comment]
cc @angelk

**Comment by angelk** [comment]
@mollieru I don't have anything in common with this project, wrong cc ;)

**Comment by mollieru** [comment]
Oops + @angel

**Comment by carodew** [comment]
Good catch @craigjh ! 

@mollieru @angel I think we should consider a non color-based indicator to tell the difference, perhaps a checkbox next to valid fields or an exclamation point in a circle for invalid fields. The positioning would just have to be in the visible range for people using screen magnifiers – so perhaps next to the label?

**Comment by mollieru** [comment]
@carodew @angel sounds like a good idea to me. 

We are already showing non-color indicators by providing the red line and pop-up text w instructions about the error. But I think additional icon to further clarify may be helpful as well. 

Here are a few examples of this in practice:

CFPB:
![screen shot 2015-08-25 at 10 38 04 am](https://cloud.githubusercontent.com/assets/3680494/9469514/71b46eba-4b15-11e5-9f6e-8a64f8ed57bb.png)

USPTO:
![screen shot 2015-08-25 at 10 39 45 am](https://cloud.githubusercontent.com/assets/3680494/9469544/a2762872-4b15-11e5-98e3-9b60ba69a45a.png)

(aslo long but helpful read on this here: http://designmodo.com/ux-form-validation/)

**Comment by maya** [comment]
@carodew yes, but we should definitely still change the green to a lighter one. For people with red-green color deficiency, what's most important with colors is that there is a noticeable difference in the lightness of the two colors. Green appears brown/beige to them. See how much lighter CFPB's green is from the red. We should do this as well.

**Comment by carodew** [comment]
@maya makes sense!

**Comment by angel** [comment]
@nickbristow our green-light (#48a463) doesn't pass the 508 color contrast but is 3.1:1. Is that ok to use for success stroke and icon color in this context? If so, then I think we should go with the green-light. Happy to add icons additional icons, @carodew if you can give some guidance and make a trello card for me that would be great! Thanks!

**Comment by nickbristow** [comment]
@angel is that color just used on the border or on the text?

**Comment by angel** [comment]
@nickbristow just for the stroke and the icon color. Like this: 
![image](https://cloud.githubusercontent.com/assets/103496/9527337/895cfdfe-4cbc-11e5-8d12-635230fa0aa6.png)

**Comment by nickbristow** [comment]
@angel yep, its perfectly fine. Color contrast only applies to text, and borders like that, i think 3:1 is perfectly fine.

**Comment by angel** [comment]
Sweet, thanks for your help @nickbristow!
